### PR TITLE
Add google cloud logger

### PR DESCRIPTION
**Pull request by nicknovitski**

### CODE REVIEW

File: src/logger/index.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import bunyan from 'bunyan';
 import bunyanDebugStream from 'bunyan-debug-stream';
 import { SentryStream } from 'bunyan-sentry-stream';
 import { Client as RavenClient } from 'raven';
+// type error when using import
+// https://github.com/googleapis/nodejs-logging-bunyan/issues/241
+const { LoggingBunyan } = require('@google-cloud/logging-bunyan');
```
Comment (dsokal) [inline src/logger/index.ts]: Could you move this line below other imports? I feel like a `require` statement doesn't belong here.

File: src/logger/index.ts [hunk 2]
```diff
@@ -65,9 +66,23 @@ if (config.logger.loggly.token) {
...
+if (config.google.credentials) {
+  const resource = {
+    type: 'generic_node',
+    labels: {
+      node_id: config.hostname,
+      location: '',
+      namespace: '',
+    }
+  };
+  streams.push(new LoggingBunyan({name: 'turtle', resource}).stream('info'));
```
Comment (dsokal) [inline src/logger/index.ts]:
```suggestion
  streams.push(new LoggingBunyan({ name: 'turtle', resource }).stream('info'));
```

File: src/logger/index.ts [hunk 3]
```diff
@@ -65,9 +66,23 @@ if (config.logger.loggly.token) {
   });
 }
 
+if (config.google.credentials) {
+  const resource = {
+    type: 'generic_node',
+    labels: {
+      node_id: config.hostname,
+      location: '',
+      namespace: '',
```
Comment (dsokal) [inline src/logger/index.ts]: Are these left empty intentionally?

Comment (nicknovitski) [reply src/logger/index.ts]: Yeah.  I'll add a comment explaining it, but all stackdriver logs are for a resource of some type, and each type has certain labels that are always present, but default to empty strings.  For `generic_node` resources, these are `project` (which is provided by the application credentials), `node_id`, `location` (ie region), and `namespace`.  I didn't have any ideas of what values to give the location and namespace, and didn't think we need them fully configurable right now, but I thought making their default values explicit was more informative.

**Pull request merged**